src/components/GiveProject/ProjectCard.tsx

```diff
@@ -570,11 +568,11 @@ export default function ProjectCard({ project, mode }: ProjectDetailsProps) {
                         className="cause-link"
                         onClick={() => handleProjectDetailsButtonClick("View Details Button")}
                       >
-                        <Button variant="contained" color="primary" className="cause-give-button">
+                        <PrimaryButton className="cause-give-button">
                           <Typography variant="h6">
                             <Trans>View Details</Trans>
                           </Typography>
```

Comment:
You can drop the typography tag here. It's not necessary w/ our button component. 
```suggestion
                            <Trans>View Details</Trans>
```